crawl-ref/source/mon-info.cc

```diff
@@ -1841,3 +1841,12 @@ const char *monster_info::pronoun(pronoun_type variant) const
     }
     return mons_pronoun(type, variant, true);
 }
+
+const bool monster_info::pronoun_plurality() const
+{
+	if (props.exists(MON_GENDER_KEY))
+	{
+		return props[MON_GENDER_KEY].get_int() == GENDER_NEUTRAL ? true : false;
```

Comment:
For conditionals like this with just a single line condition followed by a single line of of conditional code, you can skip the braces, for example:
```
if (props.exists(MON_GENDER_KEY))
    return props[MON_GENDER_KEY].get_int() == GENDER_NEUTRAL ? true : false;
```
Running util/unbrace should also catch these and fix them for you.